overlays/holo-nixpkgs/hpos-holochain-api/src/index.js

```diff
@@ -24,19 +24,35 @@ app.get('/hosted_happs', async (_, res) => {
   }
   const presentedHapps = []
   for(let i=0; i < happs.length; i++) {
-    let enabled, source_chain;
+    let app_stats, enabled, source_chains, duration, bandwidth, cpu;
     try{
-      source_chain = await callZome(appWs,`${happs[i].happ_id}::servicelogger`, 'service', 'get_source_chain_count', null)
+      // nb: servicelogger bandwidth payload is calcalated with Bytes (not bits)
+      app_stats = await callZome(appWs,`${happs[i].happ_id}::servicelogger`, 'service', 'get_happ_usage', time_interval);
       enabled = true
     } catch(e) {
+      throw new Error(`Error calling get_stats from ${happs[i].happ_id}::servicelogger : `, e);
+    }
+    
+    if (!app_stats) {
       enabled = false
-      source_chain = 0
+      source_chains = 0
+      usage = {}
+    } else {
+      ({ source_chain_count: source_chains, duration, bandwidth, cpu } = app_stats);
+      const formatted_bandwidth = formatBytesByUnit(bandwidth); // format bandwidth into object with highest appropriate unit of measurement and respective size (ie: { size: 1, unit: GB })
+      bandwidth = formatted_bandwidth;
     }
+
     presentedHapps.push({
         id: happs[i].happ_id,
         name: happs[i].happ_bundle.name,
         enabled,
-        source_chain
+        source_chains,
+        usage: {
+          duration,
+          bandwidth,
+          cpu
+        }
```

Comment:
```suggestion
        usage
```

all three suggestions go together

Reply:
Same as above. :)